Automattic/sensei
# Tweak sidebar CSS

Built on #5758 to avoid conflicts

This updates some styling around sidebar and naming, descriptions for the UI blocks, so they match how they are used across the new templates, and allows using a fixed sidebar layout in other templates.

### Changes proposed in this Pull Request

* Move CSS related to the fixed sidebar from the default theme
* Instead make it a property of the 'Two-column layout' block
* Update block descriptions accordingly
  *  Sidebar → Sidebar Menu. The main feature of this block is turning into an overlay menu on mobile
  * Two-column layout → Layout with a fixed sidebar
 

### Testing instructions

* Check that the default template, an all other templates, still look as expected
* Also check the legacy template: replace a template or create a new one with code from `themes/sensei-course-theme/templates/lesson.4-0-2.html`
